openapi.yaml

```diff
@@ -27258,6 +27668,130 @@ components:
           x-linode-cli-display: 6
           example: '2018-01-01T13:46:32'
           readOnly: true
+    PlacementGroup:
+      type: object
+      properties:
+        id:
+          type: integer
+          nullable: false
+          description: >
+            The placement group's ID. You need to provide it for all operations impacting it.
+          example: 528
+        label:
+          type: string
+          x-linode-cli-display: 1
+          x-linode-filterable: true
+          description: |
+            The unique name set for the placement group. A label has these constraints:
+
+            * It needs to begin and end with an alphanumeric character.
+            * It can only consist of alphanumeric characters, hyphens (`-`), underscores (`_`) or periods (`.`).
+          example: PG_Miami_failover
+          minLength: 1
+        region:
+          type: string
+          x-linode-cli-display: 1
+          x-linode-filterable: true
+          readOnly: true
+          description: >
+            The [region](/docs/api/regions/#regions-list) where the placement group was deployed.
+          example: us-mia
+        affinity_type:
+          type: string
+          x-linode-filterable: true
+          readOnly: true
+          description: |
+            How compute instances are distributed in your placement group. `anti-affinity:local` places compute instances in separate fault domains, but still in the same region.
+
+            **Note**: With the beta release, only `anti_affinity:local` is available for `affinity_type`.
```

Comment:
Same as above

Reply:
@Vernholio IIRC I was referring to using `x-linode-cli-display: 1` to make sure this field is displayed as a column in the CLI output